### DISCUSSION TITLE
WW3 multi.inp output

### DISCUSSION THREAD

**Discussion by fed1942** [NOAA-EMC/WW3#1053] [Q&A]
Does anyone know how to write the results in all the points of a grid? (without having to specify the coordinates of each point separately).
I was reading the manual and I saw that there is a program called ww3_ounp. However, I do not have this program installed.

**Comment by aliabdolali** [comment]
You either need to define the coordinate and use ww3_outp/ww3_ounp to post-process the binary out_pnt.ww3 or extract the results from the gridded output. The binary out_grd.ww3 should first processed using ww3_ounf/ww3_ounf and then extract at given points using python, matlab, ...

The point output is more straightforward unless you have a large number of data such as satellite tracks.
